main.py

```diff
@@ -0,0 +1,25 @@
+from src.enums.providers import Providers
+from src.providers.s3Provider import S3Provider
+from src.providers.fSProvider import FSProvider
+from src.logger.jsonLogger import Logger
+import os
+import boto3
+    
+file_provider = os.environ.get('FILE_PROVIDER', 's3')
+source = os.environ.get('SOURCE_FILE', '/source/file/path')
+destination = os.environ.get('OUTPUT_DESTINATION', 'downloads')
```

Comment:
make sure you have permissions to create this folder in open shift

Reply:
fixed in 43b9c23